README.md

```diff
@@ -33,7 +33,7 @@ fn action() -> Result<u64, ()> {
 fn main() {
     let mut core = Core::new().unwrap();
 
-    let retry_strategy = ExponentialBackoff::from_millis(10)
+    let retry_strategy = ExponentialBackoff::from_millis(10, None)
```

Comment:
Yeah, this is a good point. Given we had two more options now, `factor` and `max_delay` I decided to use the builder pattern, which still keeps things simple.